README.md

```diff
@@ -67,6 +67,20 @@ The module gives the ability to select a specific operating system on ordering o
 
 Create a configurable option named `Operating System`. Add options with the following convention: `osId|osName`. This will take priority over selected operating system in product module settings.
 
+#### Plan params
+
+The module gives the ability to select specific plan params such as `VCPU`, `Memory`, `Disk`, `Total traffic limit monthly` (for KVM and VZ), `VCPU Units`, `VCPU Limit`, `IO Priority`, `Swap` (for VZ).
+This is done in the form of configurable options.
+
+![VCPU](./docs/option-vcpu.png)
+
+Create a configurable option named one of: `VCPU`, `Memory`, `Disk`, `Total traffic limit monthly`, `VCPU Units`, `VCPU Limit`, `IO Priority`, `Swap`. Add options with the following convention: `planParamValue|planParamTitle`. This will take priority over plan params of selected plan in product module settings.
+
+`planParamValue` for `Memory` and `Swap` set in MiB.
+`planParamValue` for `Disk` set in GiB.
+`planParamValue` for `Total traffic limit monthly` set in units which are specified in total traffic limit option of selected plan in product module settings.
+`Total traffic limit monthly` option can only be used if plan has total traffic limit type and reducing network bandwidth is enabled.
```

Comment:
`Total traffic limit monthly` option can only be used if the plan selected in Module Settings for a product has Total Traffic limit enabled in SolusIO.